aries-site/src/pages/components/maskedinput.js

```diff
@@ -22,11 +34,178 @@ const MaskedInput = () => {
           <Example
             code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/MaskedInputExample/MaskedInputExample.js"
             docs="https://v2.grommet.io/maskedinput?theme=hpe#props"
+            figma="https://www.figma.com/file/fO285pXUqN9pVZ0xpRoKjL/HPE-Masked-Input-Component?node-id=1%3A89"
           >
-            <MaskedInputExample />
+            <MaskedPhoneExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            MaskedInput place syntax restrictions on the text that can be
+            entered in an input field. It is similar to{' '}
+            <Anchor label="TextInput" href={nameToPath('TextInput')} />, in that
+            is allows the user to input shorter forms of data and content.
+            However, it is used to provide extra guidance and enforcement of
+            input formats.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Usage" level={3} gap="small">
+          <SubsectionText>
+            MaskedInput should be used anytime there is formalized syntax and
+            should always be used within a{' '}
+            <Anchor label="Form" href={nameToPath('Forms')} />.
+          </SubsectionText>
+          <Text weight="bold">When using MaskedInput, you should:</Text>
+          <BulletedList
+            items={[
+              'Provide suggestions whenever possible',
+              'Restrict syntax as much as possible',
```

Comment:
"Restrict syntax" is a bit unclear. Can we supplement with an example?

Reply:
I believe this means that you should only allow valid formats -- I will rephrase the wording here